### ISSUE TITLE
Метрики: Социальная активность рейтинговых участников

### ISSUE THREAD

**Issue by cornet1973** [realtim/mmb#233]
- Количество приглашений, которыми были активированы чужие команды, и на момент активации владелец приглашения не состоит в какой либо команде.
- Количество приглашений, которыми были активированы чужие команды, и на момент активации владелец приглашения уже состоит в какой либо другой команде.
- Количество приглашений, которые протухли, хотя на момент розыгрыша лотереи их обладатели уже состояли в командах находящихся в зачёте.
- Количество приглашений, которые протухли, при том что их обладатели так и не заявились на этот ММБ в составе какой либо команды.
- Количество приглашений, которые протухли, но их обладатели заявились на ММБ (вошли в состав какой либо команды) уже после лотереи.

**Comment by leonidfishkis** [comment]
с учётом того, что "на момент"  вычислить можно не всегда.

**Comment by cornet1973** [comment]
Да, это потребует обвесить имеющиеся процедуры регистрации дополнительными проверками и триггерами.
